packages/lesswrong/lib/collections/comments/callbacks.js

```diff
@@ -29,20 +29,20 @@ const getLessWrongAccount = async () => {
   return account;
 }
 
-async function createShortformPost (comment, context) {
+async function createShortformPost (comment, currentUser) {
```

Comment:
Looks like it should work. But, the reason `new.validate` has this weird calling convention (rather than a context object) is because it's a backwards-compatibility legacy thing. The callback you want is `comments.create.validate`.